src/plugins/chat.ts

```
@@ -547,7 +547,7 @@ function createReplyTools(
         }, {
             name: 'character_reply',
             description:
```

ESLint

[error] Insert `····`

(prettier/prettier)